src/components/molecules/ImageInput/ImageInput.tsx

```diff
@@ -63,17 +70,22 @@ export const ImageInput: React.FC<ImageInputProps> = ({
     [handleFileInputChange, name, remoteUrlInputName, setValue]
   );
 
+  const onButtonClick = () => {
+    inputFileRef?.current?.click();
+  };
+
+  const containerClasses = classNames(
+    "image-input default-container",
+    {
+      containerClassName,
+      "mod--hidden": isInputHidden,
+    },
+    { disabled: loading }
```

Comment:
1. let's use 1 obj instead of 2 here.
2. can we give more specific class names following our BEM pattern?
```suggestion
    {
      containerClassName,
      "mod--hidden": isInputHidden,
      disabled: loading
    },
```

Reply:
done 4932822